runners/portability/java/src/main/java/org/apache/beam/runners/portability/PortableRunner.java

```diff
@@ -201,12 +207,16 @@ public PipelineResult run(Pipeline pipeline) {
               .setPreparationId(prepareJobResponse.getPreparationId())
               .build();
 
-      RunJobResponse runJobResponse = jobService.run(runJobRequest);
+      // Run the job and wait for a result, we don't set a timeout here because
+      // it may take a long time for a job to complete and streaming
+      // jobs currently never return a response.
+      RunJobResponse runJobResponse = jobService.withDeadline(null).run(runJobRequest);
```

Comment:
Why do we need to add `withDeadline(null)` here? Isn't it a no-op?

Reply:
You are right that we technically do not need it here because jobService was originally created without it. I added here because I wanted to explicit show that this call is made without any deadline.

I can remove it too, let me know which do you prefer.

Reply:
I prefer removing it. The comment is sufficient to show our intent here.